project_generation/content/templates/base-app/config/config.go.tmpl

```diff
@@ -15,6 +15,7 @@ type Config struct {
 	OTBatchTimeout             time.Duration `encconfig:"OTEL_BATCH_TIMEOUT"`
 	OTExporterOTLPEndpoint     string        `envconfig:"OTEL_EXPORTER_OTLP_ENDPOINT"`
 	OTServiceName              string        `envconfig:"OTEL_SERVICE_NAME"`
+	OtelEnabled                bool          `envconfig:"OTEL_ENABLED"`
```

Comment:
It looks like the code generation for the api project is taken from the base app config, as it doesn't have it's own config section unlike the event and controller projects and right now I don't really want to restructure how the code generation is done.  It's included in the specific configs for the event and controller projects and I've added it to the example README templates.

Reply:
I don't like it but there's a lot I don't like about the layered templating in the project-generation function. I think it would be easier to have just separate templates for each type. A little duplication is better than a lot of complication. Will approve as is, even if it is icky. 😆